src/Core/Auth/Services/Implementations/AuthRequestService.cs

```diff
@@ -91,6 +103,42 @@ public async Task<AuthRequest> CreateAuthRequestAsync(AuthRequestCreateRequestMo
             }
         }
 
+        // AdminApproval requests require correlating the user and their organization
+        if (model.Type == AuthRequestType.AdminApproval)
+        {
+            // TODO: When single org policy is turned on we should query for only a single organization from the current user
```

Comment:
@trmartin4 do we have a tech debt item for cleaning this up?

Reply:
Yes! https://bitwarden.atlassian.net/browse/PM-3183

Reply:
TY!